docs/en/Concept.md

```diff
@@ -0,0 +1,41 @@
+## 4 Topic
+The Topic refers to a collection of one kind message. Each topic contains several messages and one message can only belong to one topic. The Topic is the basic unit of RocketMQ for message subscription.
+## 5 Broker Server
+As the role of the transfer station, the Broker Server stores, and forwards messages.In the RocketMQ system, the Broker Server is responsible for receiving messages sent from producers, storing them and preparing to handle pull requests.It also stores message related meta data, including consumer groups, consuming progress, topics and queues info.
+## 6 Name Server
+The Name Server serves as the provider of routing service.The Producer or Consumer can find the list of Broker IP addresses corresponding each topic through name server.Multiple Name Servers can be deployed as a cluster, but are independent of each other and do not exchange information.
+## 7 Pull Consumer
+ A type of Consumer.Applications are usually pulls messages from brokers by actively calling the Consumer's pull message method,and the application has the advantages of controlling the timing and frequency of pulling messages. Once batches of messages are pulled, user application initiates consuming process.                    
+## 8 Push Consumer
+ A type of Consumer.Under this mode, after the Broker receives the data, it will actively push it to the consumer, which is generally of high real-time performance.                       
+## 9 Producer Group
+  A collection of the same type of Producer, which sends the same type of messages with consistent logic.If a transaction message is sent and the original producer crashes after sending, the Broker Server will contacts other Producer in the same Producer group to commit or rollback  the transactional message.
+## 10 Consumer Group
+  A collection of the same type of Consumer, which sends the same type of messages with consistent logic.The Consumer Group makes load-balance and fault-tolerance super easy in terms of message consuming.
+Warning: consumer instances of one consumer group must have exactly the same topic subscription(s).   
+
+RocketMQ supports two type of consumption mode:Clustering and Broadcasting.
+## 11 Consumption Mode - Clustering
+Under the Clustering mode, each Consumer instance of the same Consumer Group allocates messages equally.
+## 12 Consumption Mode - Broadcasting
+Under the Broadcasting mode, each Consumer instance of the same Consumer Group receives a full number of messages.
+## 13 Normal Ordered Message
+Under the Normal Ordered Message mode, the messages received by consumers through the same consumption queue are sequential, while the messages received by different message queues may be non-sequential.
+## 14 Strictly Ordered Message
+Under the Strictly Ordered Message mode, all messages received by the consumer are sequential.
```

Comment:
all messages received by the consumers **from the same topic** are sequential, as the order they were stored.

Seems better.